sklearn/semi_supervised/_self_training.py

```diff
@@ -36,6 +36,8 @@ class SelfTrainingClassifier(MetaEstimatorMixin, BaseEstimator):
     The classifier will continue iterating until either max_iter is reached, or
     no pseudo-labels were added to the training set in the previous iteration.
 
+    See glossary entry for :term:`metaestimators`.
+
```

Comment:
I would prefer to modified the first paragraph:

```python
This :term:`metaestimator` allows a given ...
```

Reply:
👍 Thanks for the suggestion. Addressed in the next commit.